contributors/triage/README.md

```diff
@@ -0,0 +1,13 @@
+# Kubernetes Issue Triage
```

Comment:
It should go under contributors/devel. That's the place we direct contributors from CONTRIBUTING.md files, where the index is, etc. Also, `triage` is a bad name.

Sadly, org-level projects can't be made world-readable. We have an open issue to stop using them. The contents of the link was included in my previous comment.

The most important problem to solve is scaling the effort. We used to have someone assigned to triaging incoming issues and it was not possible to keep up. We need to round-robin 1st-level triage and push 2nd-level triage to SIGs. That doesn't entirely solve the problem, since the overwhelming majority of issues go to ~6 SIGs (Node, API machinery, CLI, Networking, Storage, Apps). We may need to round-robin again within those SIGs. Just saying "somebody should triage open issues" isn't going to work.

The people who triage also need some guidance. We have previously had the problem that a significant number of issues were miscategorized. No matter what we do, I'm sure some number will get re-categorized, but keeping that number small will help issues be properly triaged in a timely manner.

Additionally, it doesn't have to be in scope for this document, but some incentive will need to be put in place to induce contributors to work on bugs as opposed to features.